### ISSUE TITLE
Slow GBA or 2600 core start, long waiting time

### ISSUE THREAD

**Issue by gitowiec** [libretro/RetroArch#11982]
## Description

It is my first time with RetroArch on a PC (the first was on my PS Vita). I installed it on Ubuntu 18.04 running on my laptop. Then I setup GBA core and 2600 core and 8bitdo controller "Zero 2" via Bluetooth (the only possible way, its USB port is only for charging). And I tried 2 games. One was Space Invaders on GBA, the second was the same on 2600. Every time I choose RUN in the menu I have to wait like 10 seconds or longer. The menu disappears, for 10 seconds there is no application window on the Ubuntu desktop and after the waiting game/core window pops up and the games begin. I run `retroarch --verbose` but can't find any suspicious activity in the log.

### Expected behavior

There is no waiting time, the core with its ROM startup instantly.

### Actual behavior

Long waiting time.

### Steps to reproduce the bug

1. Install RetroArch on Ubuntu 18.04 from official sources (PPA)
2. Do everything what I did in Description

### Bisect Results

I don't know, never run RetroArch without this lag.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.9.0-r202011022058-5e551dd92b-109~ubuntu18.04.1 

### Environment information

- OS: Ubuntu Budgie 18.04

**Comment by gitowiec** [comment]
I tried this solution but can't help to fix this bad behaviour https://github.com/libretro/RetroArch/issues/10408#issuecomment-635205564

**Comment by gitowiec** [comment]
Here is the resolution https://github.com/libretro/RetroArch/issues/10408
TL;DR; use xpadneo driver

**Comment by dfilskov** [comment]
Glad I found this :)

How / where do you switch to the xpadneo driver to get rid of the long loading times?

My Vita also goes black for ~10 secs at RA startup and afterwards when loading cores/content.

**Comment by eroldru** [comment]
same here. Also on PS Vita and the first time I installed Retroarch was very snappy and fast, sadly cannot say which version it was. After updating at some point, I believe last year, it was very slow. If anyone has a solution it would be very appreciated.

**Comment by dfilskov** [comment]
There's another issue with a god discussion about how loading assets migh cause this issu: https://github.com/libretro/RetroArch/issues/14003


But no apparent soleøueion(

**Comment by dfilskov** [comment]
There's another issue with a good discussion about how loading assets might cause this issue - or - if used - a bluetooth controller (which isn't the cause for me as I'm not using external controllers):

#14003

But no apparent solution.
